docker-machine-nfs.sh

```diff
@@ -326,7 +326,7 @@ lookupMandatoryProperties ()
   fi
 
   prop_nfshost_ip=$(VBoxManage list hostonlyifs |
-    grep "${prop_network_id}$" -A 3 | grep IPAddress |
+    grep "${prop_network_id}" -A 3 | grep IPAddress |
```

Comment:
can you elaborate this change?

Reply:
Because it unfortunately doesn't find any result.

Please see:

- Without `$`
```
jerry@JerryPC /c/Users/Jerry/projects/docker-machine-nfs (support-for-wsl) $ VBoxManage list hostonlyifs | grep "VirtualBox Host-Only Ethernet Adapter #7" -A 3                                                                 
Name:            VirtualBox Host-Only Ethernet Adapter #7
GUID:            77d39a46-7b20-4a55-94f8-13fcfa757ca9
DHCP:            Disabled
IPAddress:       192.168.99.1
--
VBoxNetworkName: HostInterfaceNetworking-VirtualBox Host-Only Ethernet Adapter #7
```
- With `$`
```
jerry@JerryPC /c/Users/Jerry/projects/docker-machine-nfs (support-for-wsl) $ VBoxManage list hostonlyifs | grep "VirtualBox Host-Only Ethernet Adapter #7$" -A 3 
```

It's because of difference of line endings in Linux (LF) and Windows (CRLF) systems. Used `VBoxManage` is a windows binary and outputs CRLF.

---

There are two ways how to handle this: 
- Either do not look for line ends - omit `$` at the end 
*or*
- Remove `\r` (CR) in console output - e.g.: `VBoxManage list hostonlyifs | tr -d '\r' | grep "VirtualBox Host-Only Ethernet Adapter #7$"`

I like the first solution more, because it seems to be a bit cleaner. What do you think?

Reply:
At the end the removing of `\r` has to be there, because it causes an issue later on.

Reply:
The problem is that removing the $ will regress a previous bug => https://github.com/adlogix/docker-machine-nfs/commit/ef23e63644113f05c8f4e9532e704a4d466a3f46

Reply:
More info on the bug => https://github.com/adlogix/docker-machine-nfs/issues/71

Reply:
I see.  So I've put the trailing `$` back and I'm removing `\r` from console output.

```
prop_nfshost_ip=$(VBoxManage list hostonlyifs | tr -d '\r' |
    grep "${prop_network_id}$" -A 3 | grep IPAddress |
    cut -d ':' -f2 | xargs);
```

Reply:
It would be cool if anybody with a mac / linux can test this as I/we are not using the library anymore :/